microsetta_public_api/utils/testing.py

```diff
@@ -146,6 +146,8 @@ class TestDatabase:
     def __init__(self, n_samples=2000, seed=None):
         np.random.seed(seed)
         sample_set = [f'sample-{i + 1:04d}' for i in range(n_samples)]
+        sample_set.append('10317.000001890')
+        n_samples = len(sample_set)
```

Comment:
```suggestion
```

I should not have hard-coded this. Removing until I can find a more flexible way to add barcodes to the `TestDatabase`.